HyperV/scripts/config.ps1

```diff
@@ -17,9 +17,7 @@ $pythonDir = "C:\Python27"
 $pythonScripts = "$pythonDir\Scripts"
 $pythonArchive = "python.zip"
 $pythonExec = "$pythonDir\python.exe"
-$openstackLogs="$openstackDir\Log"
```

Comment:
Since you're going to make some significant workflow changes, it's worth mentioning them in the commit message.